mlflow/utils/openai_utils.py

```diff
@@ -109,17 +116,45 @@ def _mock_chat_completion_response(content=TEST_CONTENT):
     return _MockResponse(200, _chat_completion_json_sample(content))
 
 
+def _chat_completion_stream_chunk(content):
```

Comment:
note: We didn't have mock for streaming, meaning that we didn't test it... (OAI client has validation for streaming chunk, which raises error for non-streaming mock response).